internal/ingress/annotations/proxy/main.go

```diff
@@ -301,9 +301,10 @@ func (a proxy) Parse(ing *networking.Ingress) (interface{}, error) {
 		config.BufferSize = defBackend.ProxyBufferSize
 	}
 
+	// Only set BusyBuffersSize if annotation is present
```

Comment:
Could you please add the documentation link here?  This way we know that leaving this blank is to let Nginx use its own default value
 http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_busy_buffers_size

Reply:
Done in ea7fa2612